src/components/FeedbackForm.jsx

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+export default function FeedbackForm() {
+  const [submitted, setSubmitted] = useState(false);
+  const [valid, setValidity] = useState(false);
+  const [formValues, setFormValues] = useState({
+    firstName: '',
+    lastName: '',
+    mail: '',
+    feedback: ''
+  });
+
+  const changeVal = e => {
+    setFormValues({ ...formValues, [e.target.id]: e.target.value });
+  };
+
+  const resetForm = data => {
+    if (data.status === 200) {
+      setFormValues({
+        firstName: '',
+        lastName: '',
+        mail: '',
+        feedback: ''
+      });
+      setSubmitted(false);
+      setValidity(false);
+    } else {
+      alert('Something went wrong! Please try again.');
+    }
+  };
+
+  const formSubmit = e => {
+    e.preventDefault();
+    setSubmitted(true);
+
+    if (formValues.firstName && formValues.lastName && formValues.mail && formValues.feedback) {
```

Comment:
you can destructure `formValues` object here

Reply:
how?

Reply:
```
const { firstName, .... } = formValues;
if (firstName && ....) {
```